Automattic/woocommerce-payments
# Release branch for 8.2.0

:warning: Please do not merge the PR from the GitHub interface. :warning:

 Instead, you can use the following command:
```
 git checkout release/8.2.0 && git pull 
 git checkout trunk && git pull 
 git merge --no-ff release/8.2.0 -m 'Merge release/8.2.0 into trunk' 
 git push origin trunk 
``` 
 Changelog: 
```
* Add - add: test instructions icon animation
* Add - Added Embdedded KYC, currently behind feature flag.
* Fix - Avoid unnecessary account data cache refresh on WooPayments pages refresh.
* Fix - Check payment method is available before rendering it.
* Fix - Disables custom checkout field detection due to compatibility issues and false positives.
* Fix - Disables testing instructions clipboard button on HTTP sites when navigator.clipboard is undefined.
* Fix - fix: missing translations on testing instructions.
* Fix - fix: platform_global_theme_support_enabled undefined index
* Fix - fix: testing instructions dark theme support
* Fix - Fix caching with tracking cookie.
* Fix - Fixed an issue where the Connect page would scroll to the top upon clicking the Enable Sandbox Mode button.
* Fix - Fixed default borderRadius value for the express checkout buttons
* Fix - Fix shipping rates retrieval method for shortcode cart/checkout.
* Fix - Fix support for merchant site styling when initializing WooPay via classic checkout
* Fix - Fix WooPay direct checkout.
* Fix - Move woopay theme support checkbox to the appearance section.
* Fix - Pass appearance data when initiating WooPay via the email input flow
* Fix - Prevent preload of BNPL messaging if minimum order amount isn't hit.
* Fix - Redirect user to WooPay OTP when the email is saved.
* Fix - Remove obsolete ApplePay warning on wp-admin for test sites.
* Fix - Update cache after persisting the User session via WooPay
* Fix - Updates test mode instructions copy for cards at checkout.
* Update - update: payment method fees in one line
* Update - Update Jetpack packages to the latest versions
* Dev - Fix failing e2e tests for saved cards.
* Dev - Fix Klarna product page message E2E test after the contents inside the iframe were updated.
* Dev - Migrate Klarna E2E tests to playwright. Reduce noise in E2E tests console output.
* Dev - Migrate multi-currency e2e tests to Playwright.
```
